prowler/providers/common/audit_info.py

```diff
@@ -236,6 +236,25 @@ def set_aws_audit_info(self, arguments) -> AWS_Audit_Info:
         if not arguments.get("only_logs"):
             self.print_audit_credentials(current_audit_info)
 
+        # Parse Scan Tags
+        input_scan_tags = arguments.get("scan_tags")
+        scan_tags = []
+        if input_scan_tags:
+            for tag in input_scan_tags:
+                key = tag.split("=")[0]
+                value = tag.split("=")[1]
+                scan_tags.append({"Key": key, "Values": [value]})
+            # Get Resources with scan_tags for all regions
+            tagged_resources = []
+            for region in current_audit_info.audited_regions:
+                client = current_audit_info.audit_session.client(
+                    "resourcegroupstaggingapi", region_name=region
+                )
+                get_resources_paginator = client.get_paginator("get_resources")
+                for page in get_resources_paginator.paginate(TagFilters=scan_tags):
+                    for resource in page["ResourceTagMappingList"]:
+                        tagged_resources.append(resource["ResourceARN"])
+            current_audit_info.audit_resources = tagged_resources
```

Comment:
Please, include this code block as a separate function a include test for this function. Thanks!